File: statistical_test/fdr_bh.R
#!/usr/bin/env Rscript
p_values = c(0.21, 0.001, 0.1, 0.06, 0.005)
p.adjust(p_values, method="fdr") # => c(0.2100, 0.0050, 0.1250, 0.1000, 0.0125)

File: statistical_test/fdr_bh.py
#!/usr/bin/env python
from statsmodels.stats.multitest import multipletests

p_values = [0.21, 0.001, 0.1, 0.06, 0.005]
print(multipletests(p_values, method='fdr_bh')[1]) # => [0.21, 0.005, 0.125, 0.1, 0.0125]
